packages/valory/skills/hello_world_abci/behaviours.py

```diff
@@ -104,10 +166,11 @@ def async_act(self) -> Generator:
         """
 
         with self.context.benchmark_tool.measure(self.behaviour_id).local():
-            keeper_address = sorted(self.synchronized_data.participants)[
-                self.synchronized_data.period_count
-                % self.synchronized_data.nb_participants
-            ]
+            participants = sorted(self.synchronized_data.participants)
+            random.seed(self.synchronized_data.most_voted_randomness, 2)  # nosec
```

Comment:
Note # nosec tag has been added in this and the next line to avoid `bandit` complain about unsecure usage of pseudo random methods.